### ISSUE TITLE
Interpreting output

### ISSUE THREAD

**Issue by chollenbeck** [ArdiaD/DEoptim#14]
Hi - I'm getting some unexpected output and I'm not sure if I'm missing something simple about the return values of the DEoptim() function. It is difficult to give a reproducible example, but this is generally what I'm seeing:

I can run DEoptim supplying my own objective function:

```
results <- DEoptim::DEoptim(my_of, ...)
```

I am trying to extract the iteration that maximizes the function like this:

```
bestmem <- results$optim$bestmem
```

However, if I run 'bestmem' through the objective function as below, the value of the objective function returned is not equal (usually quite a bit less optimal) to `results$optim$bestval`

```
my_of(results$optim$bestmem)
```

In other words: `my_of(results$optim$bestmem)` != `results$optim$bestval`

Can you help me understand why that is the case? Thanks in advance for your help.

**Comment by joshuaulrich** [comment]
I can't think of a reason how/why DEoptim could be responsible for this. It calls your objective function with the `bestmem` values and your function returns the `bestval` results.

My guess is that it's something in your objective function... a random component, perhaps? Can you share your objective function? You can email it to me if you'd prefer to keep it private.

**Comment by joshuaulrich** [comment]
Closing this because it's unlikely something we can fix in DEoptim.
